layouts/default.vue

```diff
@@ -91,7 +91,8 @@ p {
   display: flex;
   flex-direction: column;
   height: 100vh;
-  overflow: hidden; 
+  overflow-y: scroll;
```

Comment:
I think it is probably better to set overflow-y to auto, so

overflow-y: auto;

That way the scroll bar will only show if it is needed. Frees up a little bit of space (right-hand side of page) on the map pages. And just keep overflow-x as is. Thanks